app/src/main/java/net/squanchy/eventdetails/EventDetailsService.java

```diff
@@ -49,7 +49,7 @@
                     dayId,
                     apiEvent.name,
                     apiEvent.place_id,
-                    OptionalEnums.from(apiEvent.experience_level),
+                    Optional.fromNullable(apiEvent.experience_level).flatMap(ExperienceLevel::fromNullableRawLevel),
```

Comment:
...which Optional is this? The one we have from remote doesn't have `fromNullable()`. We probably want to settle on just one and ditch the other. I would drop the remote dependency at this point.

⚠️  you will need to fix usages of non-replaceable APIs (such as `ifPresentOrElse()` from #103) then stuff like this can be replaced with:

```java
Optional.fromNullable(apiEvent.experience_level)
    .map(ExperienceLevel::fromNullableRawLevel)
```

And that's it (magic of `map`ping)